packages/webpack/src/utils/with-web.ts

```diff
@@ -214,35 +214,78 @@ export function withWeb() {
       },
     ];
 
-    const rules: RuleSetRule[] = [
+    const globalStyleRules: RuleSetRule[] = [
```

Comment:
Expand global stylesheets to process less, stylus as well. It was missed previously.